packages/react-native/src/Authenticator/hooks/useFieldValues/__tests__/useFieldValues.spec.ts

```diff
@@ -82,6 +86,20 @@ describe('useFieldValues', () => {
     });
   });
 
+  it('logs a warning for non array fields', () => {
+    const mockFields = 'test';
+    // test js users usecase, ignore ts on purpose
+    //@ts-ignore
```

Comment:
Rather than introduce `@ts-ignore` in test cases, cast the invalid value, example `mockFields as unknown as RadioFieldOptions[]`

Reply:
Thanks! I also went ahead a modified another test using this pattern that slipped through.